docs/example-usage.ipynb

```diff
@@ -0,0 +1,1478 @@
+   "source": [
+    "### What is L2 Gridded data?\n",
+    "\n",
+    "Level 2 Gridded Data is data that is stored in 1D arrays along with row and column indices.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a7740dd-bed5-4ddc-aefd-4f89b47576e7",
+   "metadata": {},
+   "source": [
+    "| idx             |  0 |  1 |  2 |  3 |  4 |  5 |  6 |  7 | ... |\n",
+    "|-----------------|----|----|----|----|----|----|----|----|-----|\n",
+    "| data_variable   | 10 | 20 | 25 | 15 | 75 | 90 | 60 | 30 | ... |\n",
+    "| EASE_row_idx    |  3 |  3 |  2 |  1 |  0 |  1 |  2 |  3 | ... |\n",
+    "| EASE_column_idx |  3 |  2 |  2 |  1 |  1 |  0 |  0 |  0 | ... |\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39c88eb8-a1cf-4e55-96c0-6991b9da4caa",
+   "metadata": {},
+   "source": [
+    "### Input Data Structure\n",
+    "\n",
+    "Open the input Level 2 Gridded Data file and examine the top level groups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89c8bdfb-3fc7-4ad9-95a0-fbc722b0bce4",
+   "metadata": {},
+   "source": [
+    "On a mac you can open the input file in panoply to examine it with the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e374cfd8-a404-4f7c-81cd-50498ca82cb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!open -a panoply SMAP_L2_SM_P_00872_D_20150401T044811_R19240_001.h5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "02ff7203-a83c-422c-b867-a606dc4b9dbb",
+   "metadata": {},
+   "source": [
+    "Open with xarray and examine the groups and some data variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "eae23a84-3517-4b4d-84dc-6831d4c9276d",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Top level Groups: ['Metadata', 'Soil_Moisture_Retrieval_Data']\n",
+      "\n",
+      "First 10 data variables:\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['EASE_column_index',\n",
+       " 'EASE_row_index',\n",
+       " 'albedo',\n",
+       " 'albedo_option3',\n",
+       " 'boresight_incidence',\n",
+       " 'bulk_density',\n",
+       " 'clay_fraction',\n",
+       " 'freeze_thaw_fraction',\n",
+       " 'grid_surface_status',\n",
+       " 'landcover_class']"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import xarray as xr\n",
+    "\n",
+    "in_datatree = xr.open_datatree(\n",
+    "    'SMAP_L2_SM_P_00872_D_20150401T044811_R19240_001.h5', decode_times=False\n",
+    ")\n",
+    "groups = [c for c in in_datatree.children]\n",
+    "print(f'Top level Groups: {groups}\\n')\n",
+    "\n",
+    "data_vars = in_datatree['Soil_Moisture_Retrieval_Data'].data_vars\n",
+    "print('First 10 data variables:')\n",
+    "list(data_vars)[0:10]"
```

Comment:
More a question than a request for change - why did you do this instead of just using the `DataTree` HTML display? Was it because the attributes are automatically expanded and they have a whole bunch in them?

Reply:
because there's 53 variables in the repl and the attributes are annoying

Reply:
Well I may have changed my mind on that.